### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by SolMendiola**
## :link: Notion:

- [Exercise - TODO App](https://www.notion.so/xmartlabs/Exercise-TODO-App-7c22250bc96d440a84814e14abefabe6)

---

## :art: Design to follow:

- [Todo](https://app.zeplin.io/project/5ad6155f66fa46f3911d7211)

---

## :pencil2: Description:

Change aspects of UI 

---

### iOS

Screen Shot 2022-09-28 at 11 28 13

Screen Shot 2022-09-28 at 11 28 24

Screen Shot 2022-09-28 at 11 28 13

---

### CODE REVIEW

File: to_do_app/lib/addTask/form_to_add_new_task.dart [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'package:flutter/cupertino.dart';
```
Comment (sante) [inline to_do_app/lib/addTask/form_to_add_new_task.dart]:
```suggestion
```

File: to_do_app/lib/detailTask/detail_task.dart [hunk 2]
```diff
@@ -18,29 +18,48 @@ class DetailOfTask extends StatelessWidget {
...
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      TextButton(
+                        onPressed: null,
+                        child: Text(
+                          'MARK AS DONE',
+                          style: TextStyle(color: Colors.pinkAccent),
+                        ),
+                      ),
+                    ]),
```
Comment (sante) [inline to_do_app/lib/detailTask/detail_task.dart]: Is a row necessary as you have just one child?

Comment (sante) [reply to_do_app/lib/detailTask/detail_task.dart]: Maybe you can use `Center` instead

**Comment by sante** [comment]
Please edit the description as it has the bare template

File: to_do_app/lib/detailTask/detail_task.dart [hunk 3]
```diff
@@ -18,29 +18,48 @@ class DetailOfTask extends StatelessWidget {
...
+            Icon(Icons.arrow_back_ios, color: Colors.white),
+            Text('Todos', style: TextStyle(fontSize: 17, color: Colors.white)),
+          ]),
+        ),
+      ),
       body: Container(
-        width: 375,
-        height: 262,
+        width: double.infinity,
+//        height: 262,
```
Comment (sante) [inline to_do_app/lib/detailTask/detail_task.dart]: Remove comment

**Review by sante — APPROVED** [review]

**Review by f7deleon — APPROVED** [review]

**Pull request merged**